### ISSUE TITLE
Investigate the page view on liveblog navigation

### ISSUE THREAD

**Issue by marjisound** [guardian/dotcom-rendering#12826]
Find out what happens to page view when user changes liveblog page using the pagination & key events. We want to find out if we're getting new page views for each page, or there will be one page view throughout all page navigations

**Comment by marjisound** [comment]
So far in my investigations:

**WEB:**
On every navigation (both pagination & key events) there is a new page view id 

IOS: 
ios currently doesn't emit any ophan events when pages changes in liveblog. 
**ACTION**: we need too talk to ios about this. I've arranged a session with Dana and @arelra on Thursday 20th Nov.  

**Android:** 
Android is implementing a change to support liveblog navigation directly, opening "next pages" in a new activity/fragment without going through deeplinks. Doing this, page changes in liveblog will fire new ophan page view event

**Comment by SiAdcock** [comment]
Current behaviour: we send page view events for different pages on live blogs. When iOS do pagination, we should do the same thing.

We should check whether infinite scroll triggers page views.

**Comment by marjisound** [comment]
We found out that the web is creating new pageviews for liveblog navigations. 
In apps with the old templates, there is no pagination and instead `show more updates`  loads the older blocks in the same page. We checked ios and found that in ios is not sending any ophan events for loading the older blocks, or clicking the key events. 
@arelra has also communicated this with the Journalism Data and page.view.data.working. The email thread can be found https://groups.google.com/a/guardian.co.uk/g/dotcom.platform/c/QH1wDlxdW2E 

Actions:
- Android: is going to implement support for page navigation without going through deep linking. This change will also make it consistent with the web where changing pages in liveblog will trigger new page view events in ophan.
- ios: We need to talk to them to implement a similar behaviour that android is doing. Currently in Beta there's no ophan events being sent by any of the liveblog navigations in ios
